modules/dnn/test/test_misc.cpp

```diff
@@ -93,6 +93,42 @@ TEST(blobFromImageWithParams_4ch, NHWC_scalar_scale)
     }
 }
 
+TEST(blobFromImageWithParams_CustomPadding, letter_box)
+{
+    Mat img(40, 20, CV_8UC4, Scalar(2, 2, 2, 2));
+
+    // Custom padding value that you have added
+    Scalar customPaddingValue(1, 1, 1, 1); // Example padding value
```

Comment:
It makes sense to use different value for different channels and check each channel value.